coil-base/api/coil-base.api

```diff
@@ -660,7 +676,7 @@ public final class coil/size/OriginalSize : coil/size/Size {
 	public fun writeToParcel (Landroid/os/Parcel;I)V
 }
 
-public class coil/size/OriginalSize$Creator : android/os/Parcelable$Creator {
+public final class coil/size/OriginalSize$Creator : android/os/Parcelable$Creator {
```

Comment:
Interestingly this is not a binary incompatible change, however classes that extend this class with throw a `VerifyError` when instantiated. Assuming the Kotlin team has taken this into account.

Reply:
See [here](https://github.com/coil-kt/coil/pull/671)